pkg/cache/cache.go

```diff
@@ -48,6 +48,7 @@ func Nodes(matchingLabels map[string]string, force bool) error {
 		opts = append(opts, client.MatchingLabels(matchingLabels))
 	}
 
+	// TODO(qbarrand): use a v1.NodeList?
```

Comment:
Did you forget it?

Reply:
No, this is the actual code - I added a few TODOs to be addressed later. I did not want to test the code before we have unit tests.